src/com/mendix/widget/carousel/components/CarouselItem.ts

```diff
@@ -0,0 +1,14 @@
+import { DOM } from "react";
+
+export interface CarouselItemProps {
+    imageUrl: string;
+    active?: boolean;
+}
+
+export const CarouselItem = (props: CarouselItemProps) => {
+    return (
+        DOM.div({ className: props.active ? "item active" : "item" },
+            DOM.img({ alt: "item", src: props.imageUrl })
```

Comment:
The html required attribute should be added. We can set it with name of the image, and later if the title and / or description of an image is set via the modeler or data we can use that instead.